app/views/recipes/index.erb

```diff
@@ -0,0 +1,58 @@
+ <div class="container">
+            <div class="blog-post">
+
+                <h1>Recipes</h1>
+
+                <div class="tab">
+                        <a href="http://localhost:3000/recipes/new"><button class="tablinks" >Compose your own recipe</button><a>
```

Comment:
you shouldn't use hardcoded paths like that, please use the methods from routes

Reply:
I've deleted it for a while ... we will add links and navigation later on